src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java

```diff
@@ -69,7 +69,7 @@
   public static final String MESSAGE_WALL_QUOTE_PREFIX = "QAQuote";
 
   // links
-  public static final String EXTERNAL_LINK = "http://www.wikia.com/";
+  public static final String EXTERNAL_LINK = "http://www.pudelek.pl/";
```

Comment:
Testing `href` for `wikia.com` links on sandboxes/preview/verify would fail as we add a environment prefix to these URLs. This was a very simple solution. :trollface: